test/utils.cpp

```diff
@@ -193,6 +193,32 @@ TEST_CASE("tokenize_nl() split a string into delimiters and fields", "[utils]")
 	}
 }
 
+TEST_CASE(
+	"strip_comments returns correct part of the line",
```

Comment:
If this test fails, the programmer will see this message and might wonder what "correct" means—it just isn't saying anything. I suggest replacing it with something more specific, like "the part of the line before the first # character".

Reply:
Reworded as suggested :+1: